services/repository/src/main/java/org/sagebionetworks/repo/web/filter/JsonpCallbackFilter.java

```diff
@@ -47,6 +47,10 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
 			if(sessionToken != null){
 				throw new ServletException("JSONP callbacks are only allowed on public methods with no 'sessionToken'");
 			}
+			String accessToken = httpRequest.getHeader(AuthorizationConstants.AUTHORIZATION_HEADER_NAME);

```

Comment:
Double check with Jay that this does not break the web client

Reply:
Per Jay, this feature is most likely no longer used and therefore the change is irrelevant.